src/main/webapp/populateData.js

```diff
@@ -311,3 +312,60 @@ function populateGo() {
       });
 }
 
+/**
+ * Set up the assign panel. For now, this just prints the response
+ * from the server for /gmail-actionable-emails
+ */
+function setUpAssign() {
+  const assignContent = document.querySelector('#assign');
+
+  const subjectLinePhrases = ['Action Required', 'Action Requested'];
+  fetchActionableEmails(subjectLinePhrases, true, 7)
+      .then((response) => {
+        assignContent.innerText = response
+            .map((obj) => obj.snippet)
+            .reduce((a, b) => a + '\n' + b);
```

Comment:
Nit: I think you can use `.join("\n")` instead of the "manual" reduce

Reply:
Cool - wasn't familiar with `.join()`. Thanks!